index.js

```diff
@@ -94,10 +99,12 @@ function send(ctx, path, opts) {
       throw err;
     }
 
+    if (setHeaders) setHeaders(ctx.res, path, stats);
```

Comment:
I used "should" because it's a recommandation. The users are free to not follow it if they want to shoot themselves in the foot.

IMO it should stay there.